app/models/bookstore.rb

```diff
@@ -73,4 +75,20 @@ def self.search(search)
       Bookstore.all
     end
   end
+
+  def tweet
+    twitter_client.update("#{name}, #{city}, #{state} was just added! Check it out at: https://www.literalist.herokuapp.com/bookstores")
+  end
+
+  private
+
+  def twitter_client
+    client = Twitter::REST::Client.new do |config|
+      config.consumer_key = ENV["YOUR_CONSUMER_KEY"]
+      config.consumer_secret = ENV["YOUR_CONSUMER_SECRET"]
+      config.access_token = ENV["YOUR_ACCESS_TOKEN"]
+      config.access_token_secret = ENV["YOUR_ACCESS_SECRET"]
+    end
+    client
```

Comment:
What happens if Twitter encounters an exception? Building in a way to handle that gracefully should be included in this method.